content/renderer/brave_render_frame_impl.cc

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/content/renderer/brave_render_frame_impl.h"
+
+#include "brave/renderer/brave_content_settings_observer.h"
+#include "brave/common/network_constants.h"
+#include "brave/common/shield_exceptions.h"
```

Comment:
Above three headers in chrome(brave) layer shouldn't be included because `BraveRenderFrameImpl` is in content layer.

Reply:
We currently put both content layer and chrome layer inside brave subfolders.
I suggested to put them in chromium_src for content_layer before but @bridiver doesn't like that because that's where we use overrides.  
If we think this is important we could perhaps have chromium_src (not override) and override_chromium_src (which has override semantics). 
@bridiver thoughts?

Reply:
IMO, inserting our all new addition into `/brave` seems better.
However, we uses `/brave` as a chrome layer. So, there is no proper place our content layer implementation. Using `/brave/content` as our content layer implementation seems not bad but a little bit confusing because it seems like chrome layer includes content layer.